pkg/metrics/sketch_series_test.go

```diff
@@ -4,67 +4,68 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/DataDog/datadog-agent/pkg/config"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator/ckey"
 	"github.com/DataDog/datadog-agent/pkg/quantile"
 
 	"github.com/DataDog/agent-payload/gogen"
-	"github.com/DataDog/datadog-agent/pkg/aggregator/ckey"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestSketchSeriesListMarshal(t *testing.T) {
-	var (
-		sl         = make(SketchSeriesList, 2)
-		makesketch = func(n int) *quantile.Sketch {
-			s, c := &quantile.Sketch{}, quantile.Default()
-			for i := 0; i < n; i++ {
-				s.Insert(c, float64(i))
-			}
-			return s
-		}
+func makesketch(n int) *quantile.Sketch {
+	s, c := &quantile.Sketch{}, quantile.Default()
+	for i := 0; i < n; i++ {
+		s.Insert(c, float64(i))
+	}
+	return s
+}
 
-		// makeseries is deterministic so that we can test for mutation.
-		makeseries = func(i int) SketchSeries {
-			ss := SketchSeries{
-				Name: fmt.Sprintf("name.%d", i),
-				Tags: []string{
-					fmt.Sprintf("a:%d", i),
-					fmt.Sprintf("b:%d", i),
-				},
-				Host:     fmt.Sprintf("host.%d", i),
-				Interval: int64(i),
-			}
-
-			for j := 0; j < i+5; j++ {
-				ss.Points = append(ss.Points, SketchPoint{
-					Ts:     10 * int64(j),
-					Sketch: makesketch(j),
-				})
-			}
-
-			gen := ckey.NewKeyGenerator()
-			ss.ContextKey = gen.Generate(ss.Name, ss.Host, ss.Tags)
-
-			return ss
-		}
+// makeseries is deterministic so that we can test for mutation.
+func makeseries(i int) SketchSeries {
+	ss := SketchSeries{
+		Name: fmt.Sprintf("name.%d", i),
+		Tags: []string{
+			fmt.Sprintf("a:%d", i),
+			fmt.Sprintf("b:%d", i),
+		},
+		Host:     fmt.Sprintf("host.%d", i),
+		Interval: int64(i),
+	}
 
-		check = func(in SketchPoint, pb gogen.SketchPayload_Sketch_Dogsketch) {
-			s, b := in.Sketch, in.Sketch.Basic
-			require.Equal(t, in.Ts, pb.Ts)
-
-			// sketch
-			k, n := s.Cols()
-			require.Equal(t, k, pb.K)
-			require.Equal(t, n, pb.N)
-
-			// summary
-			require.Equal(t, b.Cnt, pb.Cnt)
-			require.Equal(t, b.Min, pb.Min)
-			require.Equal(t, b.Max, pb.Max)
-			require.Equal(t, b.Avg, pb.Avg)
-			require.Equal(t, b.Sum, pb.Sum)
-		}
-	)
+	for j := 0; j < i+5; j++ {
+		ss.Points = append(ss.Points, SketchPoint{
+			Ts:     10 * int64(j),
+			Sketch: makesketch(j),
+		})
+	}
+
+	gen := ckey.NewKeyGenerator()
+	ss.ContextKey = gen.Generate(ss.Name, ss.Host, ss.Tags)
+
+	return ss
+}
+
+func check(t *testing.T, in SketchPoint, pb gogen.SketchPayload_Sketch_Dogsketch) {
```

Comment:
You might want to call `t.Helper()` here so that when an assertion fails it is skipped and the actual test is shown instead.

Reply:
Yes, done!